src/04-modality-specific-files/10-functional-near-infrared-spectroscopy.md

```diff
@@ -0,0 +1,367 @@
+| NIRSCWOPTICALDENSITY        | Gcontinuous wave change in optical density measurements. Equivalent to dataTypeLabel dOD in SNIRF.                                                                          |
+| NIRSCWHBO                   | continuous wave oxygenated hemoglobin (oxyhemoglobin) concentration measurements. Equivalent to dataTypeLabel HbO in SNIRF.                                                 |
+| NIRSCWHBR                   | continuous wave deoxygenated hemoglobin (deoxyhemoglobin) concentration measurements. Equivalent to dataTypeLabel HbR in SNIRF.                                             |
+| NIRSCWMUA                   | continuous wave optical absorption  measurements. Equivalent to dataTypeLabel mua in SNIRF.                                                                                 |
+| ACCEL                       | Accelerometer channel, one channel for each orientation. An extra column `component` for the axis of the orientation MUST be added to the `_*channels.tsv` file (x, y or z) |
+| GYRO                        | Gyrometer channel, one channel for each orientation. An extra column `component` for the axis of the orientation MUST be added to the `_*channels.tsv` file (x, y or z)     |
+| MAGN                        | Magnetomenter channel, one channel for each orientation. An extra column `component` for the axis of the orientation MUST be added to the `_*channels.tsv` file (x, y or z) |
+| MISC                        | Miscellaneous                                                                                                                                                               |
+
+```Text
+Name         type                   source      detector      wavelength_nominal   units
+S1-D1        NIRSCWAMPLITUDE        A1          Fz            760                  V
+S1-D1        NIRSCWAMPLITUDE        A1          Fz            850                  V
+S1-D2        NIRSCWAMPLITUDE        A1          Cz            760                  V
+S2-D1        NIRSCWAMPLITUDE        A2          Fz            760                  V
+```
+
+## Optode description (`*_optodes.tsv`)
+
+{{ MACROS___make_filename_template(datatypes=["nirs"], suffixes=["optodes"]) }}
+
+File that provides the location and type of optodes. Note that coordinates MUST be
+expressed  in Cartesian coordinates according to the NIRSCoordinateSystem and
+NIRSCoordinateSystemUnits fields in `*_coordsystem.json`. If an `*_optodes.tsv`
+file is specified, a `*_coordsystem.json` file MUST be specified as well.
+The order of the required columns in the `*_optodes.tsv` file MUST be as listed below.
+
+The x, y, and z positions are for measured locations, for example with a polhemus
+digitizer. If you also have idealised positions, where you wish the optodes to be
+placed, these can be listed in the template values. SNIRF contains arrays for both
+the 3D and 2D locations of data, the BIDS format MUST store the 3D locations if
```

Comment:
```suggestion
the 3D and 2D locations of data. The BIDS format MUST store the 3D locations if
```

Reply:
This is a bit confusing. Do you need to specify 3D locations in SNIRF or just in BIDS? What's the difference between these two formats for this context?